README.md

```diff
@@ -49,6 +49,21 @@ $.ajax({
     });
 ```
 
+## Allow default to all ajax requests
+
+In this example, there is default retry to all ajax requests, for 5 seconds between retries
```

Comment:
Please update this to `Add logic to retry all failed requests with a 5 second delay`

Reply:
Fixed